bodhi/server/util.py

```diff
@@ -604,7 +604,10 @@ def cmd(cmd, cwd=None):
     if out:
         log.debug(out)
     if err:
-        log.error(err)
+        if p.returncode == 0:
+            log.info(err)
+        else:
+            log.error(err)
```

Comment:
@crungehottman ah yes, when you stack them up they kind of apply on top of each other, which counterintuitively gets them passed in reverse order. It is weird, but it makes sense if you think about how decorators work - each decorator decorates the function under it (and returns a function) - so c is applied first, then b, then a, and each tacks on its new parameter at the end of the list. So the list is ```[c, b, a]``` strangely ☺